pkg/productinfo/amazon/productinfo_amazon.go

```diff
@@ -336,12 +344,21 @@ func (e *Ec2Infoer) newGetProductsInput(regionId string) *pricing.GetProductsInp
 
 // GetRegions returns a map with available regions
 // transforms the api representation into a "plain" map
-func (e *Ec2Infoer) GetRegions(ctx context.Context) (map[string]string, error) {
-	regionIdMap := make(map[string]string)
-	for key, region := range endpoints.AwsPartition().Regions() {
-		regionIdMap[key] = region.Description()
+func (e *Ec2Infoer) GetRegions(ctx context.Context, service string) (map[string]string, error) {
+	switch service {
+	case "eks":
+		eksRegions := make(map[string]string)
+		eksRegions["us-west-2"] = "US West (Oregon)"
+		eksRegions["us-east-1"] = "US East (N. Virginia)"
+		eksRegions["eu-west-1"] = "EU (Ireland)"
```

Comment:
if it's hardcoded, then we should at least move it out to a constant/variable instead of putting it in the middle of the code

Reply:
and we can reuse the descriptions from the SDK to make sure there's no typo in them

Reply:
thanks, fixed